packages/sanity/src/core/form/components/Details.tsx

```diff
@@ -61,20 +62,20 @@ export function Details(props: DetailsProps) {
   return (
     <Box {...restProps}>
       <HeaderButton type="button" onClick={handleToggle}>
-        <Header align="center">
-          <Inline>
+        <Header>
+          <Flex align="center">
             <IconBox data-open={open ? '' : undefined}>
               <Text size={1}>
                 <ToggleArrow open={open} />
               </Text>
             </IconBox>
             {icon && <Box marginLeft={1}>{icon}</Box>}
             <Box flex={1} marginLeft={1}>
-              <Text size={1} weight="medium">
+              <Text textOverflow="ellipsis" size={1} weight="medium">
```

Comment:
not strictly related to this PR, but I noticed the AlertStrip got word breaks on narrow screens and was not rendering nicely.